pkgs/by-name/fl/fleet/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildGoModule,
+  writableTmpDirAsHomeHook,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "fleet";
+  version = "4.67.1";
+
+  src = fetchFromGitHub {
+    owner = "fleetdm";
+    repo = "fleet";
+    tag = "fleet-v${version}";
```

using finalAttrs is preferred because it doesn't introduce the entire attrset into the scope and behaves better when overridden
```suggestion
buildGoModule (finalAttrs: {
  pname = "fleet";
  version = "4.67.1";

  src = fetchFromGitHub {
    owner = "fleetdm";
    repo = "fleet";
    tag = "fleet-v${finalAttrs.version}";
```

---

Cleaner indeed